<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>275</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cocineros (Platos Fuertes)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Intervalo de segundos para generar personas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>219</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesas</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineaCantMesas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>242</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineaCantMeseros">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>187</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineaCocineros">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>298</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>164</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Meseros</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/imagenes/configuration.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>453</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myanmar Text</family>
     <pointsize>14</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Indique las cantidades respectivas por crear.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>161</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLineEdit" name="lineaTiempoGen1">
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineaTiempoGen2">
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineaCantMeseros</tabstop>
  <tabstop>lineaCantMesas</tabstop>
  <tabstop>lineaCocineros</tabstop>
  <tabstop>lineaTiempoGen1</tabstop>
  <tabstop>lineaTiempoGen2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
